src/stories/accidenz_commons.stories.js

```diff
@@ -9,4 +9,114 @@ export const demo = () => `
   <p class="accidenz-commons">
     The quick brown fox jumps over the lazy dog.
   </p>
-`
+
+`;
```

Comment:
We follow [JavaScript Standard Style](https://standardjs.com/) which says semicolons are not required.

Reply:
Okay will update the following